### PR TITLE
replace img with sizes to picture element

### LINKED ISSUE #143: [BUG] Images always loads only one size for different devices
https://www.loom.com/share/14d8d710e3a94e83be307b390b0f8d3c
https://www.loom.com/share/6f914a2aa5d64a2783b7963d72e97d2a

Great summary https://www.builder.io/blog/fast-images

```html

<picture>
  <source media="(max-width: 799px)" srcset="elva-480w-close-portrait.jpg" />
  <source media="(min-width: 800px)" srcset="elva-800w.jpg" />
  <img src="elva-800w.jpg" alt="Chris standing up holding his daughter Elva" />
</picture>

```

## Notes
1. we should consider using pictures
2. add JPEG versions (optional on this stage)
3. Medium responsive attributes look good, only need 2 versions: desktop, mobile
4. Use resize fit strategy
```html
<picture>
  <source
    srcset="https://miro.medium.com/v2/resize:fit:640/format:webp/1*zrJDoogUHHBtosKsx7afOg.png 640w, https://miro.medium.com/v2/resize:fit:720/format:webp/1*zrJDoogUHHBtosKsx7afOg.png 720w, https://miro.medium.com/v2/resize:fit:750/format:webp/1*zrJDoogUHHBtosKsx7afOg.png 750w, https://miro.medium.com/v2/resize:fit:786/format:webp/1*zrJDoogUHHBtosKsx7afOg.png 786w, https://miro.medium.com/v2/resize:fit:828/format:webp/1*zrJDoogUHHBtosKsx7afOg.png 828w, https://miro.medium.com/v2/resize:fit:1100/format:webp/1*zrJDoogUHHBtosKsx7afOg.png 1100w, https://miro.medium.com/v2/resize:fit:2000/format:webp/1*zrJDoogUHHBtosKsx7afOg.png 2000w"
    sizes="(min-resolution: 4dppx) and (max-width: 700px) 50vw, (-webkit-min-device-pixel-ratio: 4) and (max-width: 700px) 50vw, (min-resolution: 3dppx) and (max-width: 700px) 67vw, (-webkit-min-device-pixel-ratio: 3) and (max-width: 700px) 65vw, (min-resolution: 2.5dppx) and (max-width: 700px) 80vw, (-webkit-min-device-pixel-ratio: 2.5) and (max-width: 700px) 80vw, (min-resolution: 2dppx) and (max-width: 700px) 100vw, (-webkit-min-device-pixel-ratio: 2) and (max-width: 700px) 100vw, 1000px"
    type="image/webp">
  <source data-testid="og"
          srcset="https://miro.medium.com/v2/resize:fit:640/1*zrJDoogUHHBtosKsx7afOg.png 640w, https://miro.medium.com/v2/resize:fit:720/1*zrJDoogUHHBtosKsx7afOg.png 720w, https://miro.medium.com/v2/resize:fit:750/1*zrJDoogUHHBtosKsx7afOg.png 750w, https://miro.medium.com/v2/resize:fit:786/1*zrJDoogUHHBtosKsx7afOg.png 786w, https://miro.medium.com/v2/resize:fit:828/1*zrJDoogUHHBtosKsx7afOg.png 828w, https://miro.medium.com/v2/resize:fit:1100/1*zrJDoogUHHBtosKsx7afOg.png 1100w, https://miro.medium.com/v2/resize:fit:2000/1*zrJDoogUHHBtosKsx7afOg.png 2000w"
          sizes="(min-resolution: 4dppx) and (max-width: 700px) 50vw, (-webkit-min-device-pixel-ratio: 4) and (max-width: 700px) 50vw, (min-resolution: 3dppx) and (max-width: 700px) 67vw, (-webkit-min-device-pixel-ratio: 3) and (max-width: 700px) 65vw, (min-resolution: 2.5dppx) and (max-width: 700px) 80vw, (-webkit-min-device-pixel-ratio: 2.5) and (max-width: 700px) 80vw, (min-resolution: 2dppx) and (max-width: 700px) 100vw, (-webkit-min-device-pixel-ratio: 2) and (max-width: 700px) 100vw, 1000px">
  <img alt="Graph from Google Search console" class="m wo yt c" width="1000" height="390" loading="eager"
       src="https://miro.medium.com/v2/resize:fit:2000/1*zrJDoogUHHBtosKsx7afOg.png">
</picture>
```

### PR DESCRIPTION
**Pull request by AucT**
Closes #143

## Summary by CodeRabbit

- **New Features**
	- Enhanced image rendering across various templates for improved responsiveness by utilizing the `<picture>` element and removing the `sizes` parameter from image calls.
  
- **Bug Fixes**
	- Resolved inconsistencies in image rendering by standardizing the parameters passed to image partials.

### CODE REVIEW

File: themes/beaver/layouts/partials/img/404.html [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {{ $image := .image }}
 {{ $title := .title }}
-{{ partial "img/generic.html" (dict "image" $image "title" $title "widths" (slice 300 768 802) "sizes" "(max-width: 802px) 100vw, 802px" "attributes" "loading=\"lazy\" decoding=\"async\" itemprop=\"image\"") }}
+{{ partial "img/generic.html" (dict "image" $image "title" $title "widths" (slice 300 768 802) "attributes" "loading=\"lazy\" decoding=\"async\" itemprop=\"image\"") }}
```
[the remark this answers is not in the document]
Comment (pftg) [reply themes/beaver/layouts/partials/img/404.html:3]: @AucT any updates on this?

**Comment by pftg** [comment]
@AucT please give some screenshots or screencast to prove that now it's working

**Comment by AucT** [comment]
R8NmJhEoIW

fmDNMhCIEy
mcwBjboiLj
VLxEndZt7c
pM37m9mymE

**Review by pftg — CHANGES_REQUESTED** [review]
@AucT 
image

**Pull request merged**